src/clusterfuzz/_internal/bot/tasks/setup.py

```diff
@@ -257,7 +261,7 @@ def setup_testcase(testcase, job_type, setup_input, metadata=None):
     _set_timeout_value_from_user_upload(testcase_id, metadata)
 
   # Update the fuzzer if necessary in order to get the updated data bundle.
-  if setup_input.fuzzer_name:
+  if setup_input.HasField("fuzzer_name"):
```

Comment:
An empty fuzzer name is probably not valid, right? Can we just do:

```suggestion
  if setup_input.fuzzer_name:
```